### ISSUE TITLE
Register and download link doesn't work on OpenVino

### ISSUE THREAD

**Issue by bessszilard** [opencv/dldt#312]
I to follow the download instructions from a [youtube tutorial](https://www.youtube.com/watch?v=jbqOa8DX7Jg), and after clicking to the [Register and download button](https://software.seek.intel.com/openvino-toolkit?os=linux) the web page doesn't show properly.
I attach the screenshot.
![Screenshot from 2019-11-05 14-32-04](https://user-images.githubusercontent.com/48211034/68216844-7c4e4080-ffd9-11e9-9173-6e972d9b50c8.png)

I attempted to open the link in Firefox 70.0 (64-bit) and in Chromium (Version 77.0.3865.90), and as well.

**Comment by brmarkus** [comment]
Can you try via "https://software.intel.com/en-us/openvino-toolkit", please?

**Comment by bessszilard** [comment]
Thank you @brmarkus for your response.
I checked the https://software.intel.com/en-us/openvino-toolkit page already.
On the page, I clicked on the "Choose & Download" button, which lead me to the [Choose the Best Option](https://software.intel.com/en-us/openvino-toolkit/choose-download) page. I selected the Linux operating system, which led me to the ["Free Download"](https://software.intel.com/en-us/openvino-toolkit/choose-download/free-download-linux) page. On that page, I clicked the "Register & Download" button, and finally, I arrived at the not functional webpage, which I mentioned in my previous post.
